### PR TITLE
goolabs: Upgrade to v0.2.2

### PR DESCRIPTION
**Pull request by tell-k**
Upgrade to v0.2.0. Please feel free to merge :)

### CODE REVIEW

File: Library/Formula/goolabs.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Goolabs < Formula
   desc "Command-line tool for morphologically analyzing Japanese language"
   homepage "https://pypi.python.org/pypi/goolabs"
-  url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.1.1.tar.gz"
-  sha1 "3201102ab0fad77d57a09dd7c879f6f163e05799"
+  url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.2.0.tar.gz"
+  sha1 "c544460b51ad63c7d79b0ef542816c2981fa22f9"
```
Comment (mikemcquaid) [inline Library/Formula/goolabs.rb]: Mind updating all these `sha1` to `sha256`? Thanks!

Comment (tell-k) [inline Library/Formula/goolabs.rb]: @mikemcquiad  Sorry, I can't understand you. Did you say that I should change all sha1 to sha256?  

is this ok?
```
url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.2.0.tar.gz"
sha256 "34edd8b627322f39e9af0c220a196c493782e15fc302695d3701b4e3fbcc0101"  
```

thx

Comment (mikemcquaid) [inline Library/Formula/goolabs.rb]: Yep and for the `resource`s below too, thanks!

Comment (tell-k) [inline Library/Formula/goolabs.rb]: @mikemcquaid 
I have just fixed it. and upgrade goolabs new minor version 0.2.2.
Please feel free to check again :)

thanks!

File: Library/Formula/goolabs.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Goolabs < Formula
   desc "Command-line tool for morphologically analyzing Japanese language"
   homepage "https://pypi.python.org/pypi/goolabs"
-  url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.1.1.tar.gz"
-  sha1 "3201102ab0fad77d57a09dd7c879f6f163e05799"
+  url "https://pypi.python.org/packages/source/g/goolabs/goolabs-0.2.2.tar.gz"
+  sha256 "9078bf8d26b69860cf6b9f1e6143014a596ee122743130b082ba2daea8a10e26"
   revision 1
```
Comment (DomT4) [inline Library/Formula/goolabs.rb]: Please remove this line on version updates, Thanks!

Comment (tell-k) [inline Library/Formula/goolabs.rb]: @DomT4 I have removed line "revision 1". Please check it again. Thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/ddbdb61ab214932a98e34393322a1f9340c800c3. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Just a nit, but ideal commit message for version updates would be styled `goolabs 0.2.2` or similar, Thanks!

**Comment by tell-k** [comment]
@DomT4  Thanks for merge it and your advice about commit message :)

**Pull request closed without merging**